manifest_staging/chart/gatekeeper-operator/templates/gatekeeper.yaml

```diff
@@ -457,6 +457,8 @@ spec:
       release: '{{ .Release.Name }}'
   template:
     metadata:
+      annotations: 
+{{- toYaml .Values.podAnnotations | nindent 8 }}
```

Comment:
this is already in https://github.com/open-policy-agent/gatekeeper/blob/master/chart/gatekeeper-operator/templates/gatekeeper.yaml#L469 but it's missing here